chain/consensus/mir/config.go

```diff
@@ -56,13 +60,25 @@ type Config struct {
 	Consensus *ConsensusConfig
 }
 
+func DefaultConsensusConfig() *ConsensusConfig {
```

Comment:
Yes, this is nice, that's what I meant :-)

(Just resolve this comment yourself)